actions.go

```diff
@@ -272,6 +272,13 @@ func (c *Action) Fatalf(msg string, args ...interface{}) {
 	os.Exit(1)
 }
 
+// Infof prints a info-level message. The arguments follow the standard Printf
+// arguments.
+func (c *Action) Infof(msg string, args ...interface{}) {
+	// ::info <c.fields>::<msg, args>
+	fmt.Fprintln(c.w, msg, args)
```

Comment:
This should be `c.IssueCommand` like the other calls.

Reply:
I did it at first, but after looking to SDK, it's not using issue for "info".
But I have to update comment

Reply:
I was under the impression that the "old" way was deprecated

Reply:
I really have no idea.
All examples use standard output